### PR TITLE
Fix findbugs issues

### PR DESCRIPTION
**Pull request by rsandell**
@reviewbybees

### CODE REVIEW

**Comment by recena** [comment]
:bee:

**Comment by stephenc** [comment]
:bee:

File: docker-traceability-plugin/src/main/java/org/jenkinsci/plugins/docker/traceability/fingerprint/DockerInspectImageFacet.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public long getReportTimeInSeconds() {
     
     private void updateData(@Nonnull InspectImageResponse data, long reportTimeInSeconds,
             @CheckForNull String imageName) throws IOException {
-        if (reportTimeInSeconds > reportTimeInSeconds) {
+        if (reportTimeInSeconds > this.reportTimeInSeconds) {
```
Comment (stephenc) [inline docker-traceability-plugin/src/main/java/org/jenkinsci/plugins/docker/traceability/fingerprint/DockerInspectImageFacet.java]: always good when findbugs catches real bugs ;-)

**Comment by rsandell** [comment]
@reviewbybees done

File: docker-traceability-plugin/src/main/java/org/jenkinsci/plugins/docker/traceability/fingerprint/DockerInspectImageFacet.java (hunk 1 above)
Comment (oleg-nenashev) [inline docker-traceability-plugin/src/main/java/org/jenkinsci/plugins/docker/traceability/fingerprint/DockerInspectImageFacet.java]: crap :(

**Comment by oleg-nenashev** [comment]
:bee:

**Comment by oleg-nenashev** [comment]
I hope to kick-off the bugfix release soon

**Pull request merged**